soc_049_water_conflict_map/README.md

```diff
@@ -0,0 +1,19 @@
+## Water Conflict Map Dataset Near Real-time Script
+This file describes the near real-time script that retrieves and processes the [the World Water Conflict Chronology Map](https://www.worldwater.org/water-conflict/ ) for [display on Resource Watch](https://resourcewatch.org/data/explore/24928aa3-28d3-457c-ad2a-62f3c83ef663).
+
+The source provided the data as a txt file.
+
+The 'start' and 'end' columns were renamed to be 'start_year' and 'end_year' since 'end' is a reserved word in PostgreSQL. The start and end year of the conflicts were converted to datetime objects and stored in two new columns 'start_dt' and 'end_dt'. The 'source' column was renamed to be 'sources'. The resulting table was uploaded to Carto.
```

Comment:
```suggestion
The 'Start' and 'End' columns were renamed to be 'start_year' and 'end_year' since 'end' is a reserved word in PostgreSQL. The start and end year of the conflicts were converted to datetime objects (using first day of January to fill day and month for each conflict) and stored in two new columns 'start_dt' and 'end_dt'. The 'Conflict Type' column was renamed to be 'conflict_type'. The resulting table was uploaded to Carto.
```